background-charm-service/src/utils.ts

```diff
@@ -45,3 +45,27 @@ export function isValidDID(did: string): boolean {
 export function isValidCharmId(id: string): boolean {
   return !!id && id.length === 59;
 }
+
+// Derives the identity configured for this service,
+// receiving an `IDENTITY` and `OPERATOR_PASS` from the environment.
+//
+// First, uses the key path to load a key.
+// If not set, falls back to operator pass to
+// use an insecure passphrase.
+// This fallback should be removed once fully migrated
+// over to using keyfiles. 
+export async function getIdentity(identityPath?: string, operatorPass?: string): Promise<Identity> {
+  if (identityPath) {
+    console.log(`Using identity at ${identityPath}`);
+    try {
+      const pkcs8Key = await Deno.readFile(identityPath);
+      return await Identity.fromPkcs8(pkcs8Key);
+    } catch (e) {
+      throw new Error(`Could not read key at ${identityPath}.`);
+    }
+  } else if (operatorPass) {
+    console.warn("Using insecure passphrase identity.");
```

Comment:
for toolshed we have an env called ENVIRONMENT - which we can set to production.  I wonder if we should do that pattern as well here?

@jakedahn / @jsantell thoughts on unifying this idea?